### PR TITLE
Added links to manual in the context of concepts syntax changes

### PR DESCRIPTION
**Pull request by jrbeaumont**
@snowleopard What do you think?

### CODE REVIEW

**Comment by snowleopard** [comment]
I think instead of `The tool however needs to use slightly different syntax, which is explained in the manual, included in the tool and available from` it would be better to say exactly what is different:

Note that in Haskell we cannot have postfix operators `a+` and `a-`, and therefore the rising and falling transitions are denoted by `rise a` and `fall a` instead.

Much more direct, and no need to read any other documents!

**Comment by jrbeaumont** [comment]
I will apply this to all the posts, just so someone doesn't have to read from the beginning if they want to get stuck in to concepts (hopefully haha)

**Comment by snowleopard** [comment]
Also worth adding: 

We use `a+` and `a-` in this blog series for better readability and closer correspondence with STGs.

File: _posts/2015-09-10-concepts.md [hunk 1]
```diff
@@ -35,10 +35,12 @@ A tool to translate concepts to STGs is available from <https://github.com/tuura
 This uses an algorithm to translate concepts, which will be explained in this
 series of posts. Instruction on how to use the tool and examples of concept
 specifications can be found here. All concepts displayed in this series is
-can be used with the tool. and are directly translatable to an STG. The tool
-however needs to use slightly different syntax, which is explained in the
-manual, included in the tool and available from
-<https://jrbeaumont.github.io/concepts-manual/manual.pdf>
+can be used with the tool, and are directly translatable to an STG. Note that
+in this series, we use a+ and a- for better readability and closer
```
Comment (snowleopard) [inline _posts/2015-09-10-concepts.md]: `a+` and `a-`

**Comment by snowleopard** [comment]
The rest is fine!

**Pull request merged**